pkg/utils/utils.go

```diff
@@ -0,0 +1,43 @@
+package utils
+
+import (
+	openlibertyv1beta1 "github.com/OpenLiberty/open-liberty-operator/pkg/apis/openliberty/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+// Utility methods specific to Liberty and it's configuration
+
+// CustomizeEnv adds configured env variables appending configured liberty settings
+func CustomizeEnv(pts *corev1.PodTemplateSpec, la *openlibertyv1beta1.LibertyApplication) {
+	targetEnv := la.GetEnv()
```

Comment:
This should already have been set by CustomizePodSpec and pts object should contain user defined Env already
we only care about appending extra vars to pts

Reply:
Good point, got rid of targetEnv entirely and all logic relates to the consoleFormat env var now